package.json

```diff
@@ -13,7 +13,9 @@
     ".": {
       "import": "./dist/vue-stripe-menu.es.js",
       "require": "./dist/vue-stripe-menu.umd.js"
-    }
+    },
+    "./css": "./dist/vue-stripe-menu.css",
```

Comment:
Thanks for the PR.

I think we should leave current import for backward compatibility.

These imports should work just as they did:

```
vue-stripe-menu/src/scss/index
vue-stripe-menu/dist/vue-stripe-menu.css
etc
```

But I don't really know what it should look like.

```
{
  "./dist/*": "./dist/*",
  "./src/*": "./src/*"
}
```

Reply:
```json
"./dist/*": "./dist/*",
"./src/*": "./src/*"
```

Works too btw, but I guess it's not recommended.